### PR TITLE
Bugfix paligemma-client

### PR DESCRIPTION
**Pull request by probicheaux**
# Description

Fix IMAGE_PATH in client

## Type of change

Please delete options that are not relevant.

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

local

## Any specific deployment considerations

no

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

File: examples/paligemma/paligemma_client.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import os
 
 PORT = 9001
-API_KEY = os.environ["API_KEY"]
+# API_KEY = os.environ["API_KEY"]
+API_KEY = "yqoRhbDSjh25rkRRkmbL"
```
Comment (yeldarby) [inline examples/paligemma/paligemma_client.py:7]: Need to revoke

**Comment by PawelPeczek-Roboflow** [comment]
Could u also take a look at docs build failing on new modules https://github.com/roboflow/inference/actions/runs/9099684944/job/25012807735

**Comment by PawelPeczek-Roboflow** [comment]
Could u inject the key into GHA secrets?

**Comment by probicheaux** [comment]
This is an example script to be used as a client

**Comment by PawelPeczek-Roboflow** [comment]
yeah, but I guess u revoked key and now CI fails on 403

**Comment by PawelPeczek-Roboflow** [comment]
I do not have access to the project

**Comment by probicheaux** [comment]
> yeah, but I guess u revoked key and now CI fails on 403

Oh, I see now. Yep will fix

**Review by PawelPeczek-Roboflow — APPROVED** [review]
LGTM

**Pull request merged**